leaf-ui/js/loadSaveFunctions.js

```diff
@@ -263,6 +263,10 @@ function loadFromObject(obj) {
 	model.constraints = getConstArr(obj.model.constraints);
 	model.maxAbsTime = obj.model.maxAbsTime;
 
+	// store deep copy of model for detecting model changes
```

Comment:
Not sure where to put this comment, but I found a bug, I'm not sure what is causing it. If I use the bikebefore.json test file from our repos this functionality doesn't work. I'm not sure if thats because there are no actors or something strange like that, but worth taking a look into

Reply:
Would love to chat more about this bug, because I haven't been able to recreate it!